python/python/tests/test_transaction.py

```diff
@@ -291,3 +291,14 @@ def test_transaction_clear(self):
         transaction.select(1)
         transaction.clear()
         assert len(transaction.commands) == 0
+
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_transaction_chaning_calls(self, redis_client: TRedisClient):
```

Comment:
you don't need to write a test to handle this. Just change the existing transaction test to use chaining.

Reply:
the current one cant use chaining because we modified it to be more readable